Intersect.Client/Interface/Shared/SettingsWindow.cs

```diff
@@ -812,6 +861,33 @@ private void SettingsApplyBtn_Clicked(Base sender, ClickedEventArgs arguments)
                 Globals.Database.TargetFps = newFps;
             }
 
+            var newCameraShakeIntensity = 0;
+            if (mCameraShakeList.SelectedItem.Text == Strings.Settings.CameraShakeIntensityNone)
```

Comment:
Couldn't you just use the selected index and then add/subtract to get the correct value?

That said, I almost think this should be a number slider instead.